daphne_worker/src/lib.rs

```diff
@@ -14,6 +14,12 @@ use prio::codec::{Decode, Encode};
 use serde::{Deserialize, Serialize};
 use worker::*;
 
+/// Error Parameters used by the Leader to report errors caused by peers.
+#[derive(Deserialize, Serialize)]
+pub(crate) enum PeerErr {
+    ErrBatchOverlap,
+}
```

Comment:
I think I see what you're getting at here: There may be other reasons for a DO to signal an abort, and this would be the enum that defines them. However I think trying to be general here is cumbersome and makes internal APIs a little awkward. My preference would be to do away with this enum here and modify the enum in aggregate_store.rs as follows:

```rust
#[derive(Deserialize, Serialize)]
pub(crate) enum AggregateStoreResult {
    Ok(DapAggregateShare),
    ErrBatchOverlap,
}
```

This saves the caller of the AggregateStore DO from having to handle undefined behavior (e.g., suppose AggregateStore returned an error variant other than ErrBatchOverlap).

Reply:
I thought it would be nice to have a separate enum, but your point makes sense.
I've updated the enum.